content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,144 @@
+---
+title: 'Spectro Kubernetes Dashboard'
+metaTitle: 'Spectro Kubernetes Dashboard'
+metaDescription: 'Kubernetes Dashboard Monitoring pack in Spectro Cloud'
+hiddenFromNav: true
+type: "integration"
+hideToC: false
+category: ['Spectro Integrations']
```

Comment:
```suggestion
category: ['Spectro Integrations', 'monitoring']
```
We need both categories as in the UI today, users will find it through the monitoring category.